Sources/Clients/BackupsClient/BackupsClient+Test.swift

```diff
@@ -16,7 +16,10 @@ extension BackupsClient: TestDependencyKey {
 		lookupProfileSnapshotByHeader: unimplemented("\(Self.self).lookupProfileSnapshotByHeader"),
 		importProfileSnapshot: unimplemented("\(Self.self).importProfileSnapshot"),
 		importCloudProfile: unimplemented("\(Self.self).importCloudProfile"),
-		loadDeviceID: unimplemented("\(Self.self).loadDeviceID")
+		loadDeviceID: unimplemented("\(Self.self).loadDeviceID"),
+		reclaimProfileOnThisDevice: unimplemented("\(Self.self).reclaimProfileOnThisDevice"),
```

Comment:
We display an alert in `Main` informing user that their profile might be used on another device, in which the can select "reclaim" or "stop using" (not that copy, see copy further down).